Backend/Gmfctn/Controllers/UserController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using AutoMapper;
+using Data_;
+using Data_.Dtos;
+using Data_.Entities;
+using Data_.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Gmfctn.Controllers
+{
+    [Route("api/user")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private IUnitOfWork unitOfWork;
+        private readonly IMapper _mapper;
+        public UserController(GmfctnContext context, IMapper mapper)
+        {
+            unitOfWork = new UnitOfWork(context);
+            _mapper = mapper;
+        }
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<User>>> GetAllAchievement()
+        {
+            IEnumerable<User> Items = null;
```

Comment:
Please follow one naming convention. Or Pascal case, or Camel case. Code will be more clear)